shared/src/validation-schemas/interview/interview-update.validation-schema.ts

```diff
@@ -7,6 +7,7 @@ const interviewUpdate = Joi.object({
   [getNameOf<InterviewsUpdateRequestDto>('interviewerUserId')]: Joi.number()
     .integer()
     .required(),
+  [getNameOf<InterviewsUpdateRequestDto>('status')]: Joi.string().required(),
```

Comment:
I think if all status options are already defined, then status field should be checked to be one of them inside validation